recipe/meta.yaml

```diff
@@ -38,9 +38,9 @@ test:
     - pytest
   imports:
     - numcodecs
-    - numcodecs.blosc
-    - numcodecs.lz4
-    - numcodecs.zstd
+    - numcodecs.blosc  # [python_impl == "pypy"]
+    - numcodecs.lz4    # [python_impl == "pypy"]
+    - numcodecs.zstd   # [python_impl == "pypy"]
```

Comment:
Skipping these `import` tests on PyPy as we [don't build them on PyPy]( https://github.com/zarr-developers/numcodecs/blob/b7c5a81407422b00a16b6eba3c63def335be51d5/setup.py#L348-L349 ). More details in issue ( https://github.com/zarr-developers/numcodecs/issues/238 ).